examples/nlp/language_modeling/bert_pretraining.py

```diff
@@ -18,7 +18,8 @@
 To pretrain BERT on raw text dataset run
 python bert_pretraining.py \
 --amp_opt_level "O0" \
---data_dir data/lm/wikitext-2 \
+--train_data data/lm/wikitext-2/train.txt \
```

Comment:
could you please change it to --train_data path_to/wikitext-2/train.txt and --eval_data path_to/wikitext-2/valid.txt ?
the user now can choose where to download the data, data/lm are gone

Reply:
done